### PR TITLE
Adapt release.sh to update the latest version in the debugging docs.

### PR DESCRIPTION
**Pull request by Airblader**

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -147,6 +147,7 @@ git add downloads/i3-${RELEASE_VERSION}.tar.bz2*
 cp ${TMPDIR}/i3/RELEASE-NOTES-${RELEASE_VERSION} downloads/RELEASE-NOTES-${RELEASE_VERSION}.txt
 git add downloads/RELEASE-NOTES-${RELEASE_VERSION}.txt
 sed -i "s,<h2>Documentation for i3 v[^<]*</h2>,<h2>Documentation for i3 v${RELEASE_VERSION}</h2>,g" docs/index.html
+sed -i "s,Verify you are using i3 ≥ .\..,Verify you are using i3 ≥ ${RELEASE_VERSION},g" docs/debugging.html
```
Comment (stapelberg) [inline release.sh]: The regexp is broken for 4.10 and above. Can you change it to just match everything until the end of the line instead?

File: docs/debugging [hunk 2]
```diff
@@ -12,8 +12,8 @@ process and/or need further help, do not hesitate to contact us!
 
 == Verify you are using i3 ≥ 4.7
```
Comment (stapelberg) [inline docs/debugging]: While you’re touching this, please replace 4.7 by 4.10

**Comment by Airblader** [comment]
@stapelberg Done.

**Comment by Airblader** [comment]
@stapelberg Done.

**Pull request merged**